File: FootScout-Vue.WebClient/src/models/dtos/PlayerAdvertisementCreateDTO.ts
import type { SalaryRangeCreateDTO } from "./SalaryRangeCreateDTO";

// Modeluje ogłoszenie zawodnika, uwzględniając dane o pozycji, lidze, wieku, wzroście, 
// preferowanej nodze i przedziale wynagrodzenia.

export interface PlayerAdvertisementCreateDTO {
    playerPositionId: number;
    league: string;
    region: string;
    age: number;
    height: number;
    playerFootId: number;
    salaryRangeDTO: SalaryRangeCreateDTO;
    playerId: string;
}

File: FootScout-Vue.WebAPI/Models/DTOs/PlayerAdvertisementCreateDTO.cs
using FootScout_Vue.WebAPI.Entities;

namespace FootScout_Vue.WebAPI.Models.DTOs
{
    // Model DTO dla tworzenia ogłoszenia piłkarskiego
    public class PlayerAdvertisementCreateDTO
    {
        public int PlayerPositionId { get; set; }
        public string League { get; set; }
        public string Region { get; set; }
        public int Age { get; set; }
        public int Height { get; set; }
        public int PlayerFootId { get; set; }
        public SalaryRangeDTO SalaryRangeDTO { get; set; }
        public string PlayerId { get; set; }
    }
}